### ISSUE TITLE
Is google deprecating Third party cookies on December 27th 2024 ?

### ISSUE THREAD

**Issue by saisudhirpounjula** [privacysandbox/privacy-sandbox-dev-support#457]
Referring to: https://privacysandbox.com/news/privacy-sandbox-update/

It is not clear whether Google Chrome is restricting third party cookies. Please let me know if Google Chrome will absolutely restrict third party cookies on December 27th 2024 ?

**Comment by krgovind** [comment]
No, Chrome will not be rolling out third-party cookie restrictions on Dec 27th. Instead, as the announcement you linked to says, we are currently developing a new approach that would elevate user choice - offering users an informed choice about third-party cookies. Please follow along on https://privacysandbox.com/intl/en_us/news/ for updates on timeline, but we do not have further information to share at this time.
